lib/model/QubitInformationObject.php

```diff
@@ -3200,14 +3200,10 @@ public function getDigitalObjectLink()
    */
   private function generateSlug()
   {
-    if (null === $slugBasis = QubitSetting::getByName('slug_basis_informationobject'))
+    if (null === $slugBasis = sfConfig::get('app_slug_basis_informationobject'))
     {
       $slugBasis = QubitSlug::SLUG_BASIS_TITLE; // Fall back to title as slug basis
     }
-    else
-    {
-      $slugBasis = $slugBasis->getValue();
-    }
```

Comment:
`sfConfig::get()` accepts a default value, so you could replace the entire `if` with:

`$slugBasis = sfConfig::get('app_slug_basis_informationobject', QubitSlug::SLUG_BASIS_TITLE);`